extensions/wikia/PortableInfobox/services/PortableInfoboxMobileRenderService.class.php

```diff
@@ -136,12 +140,31 @@ private function renderInfoboxHero( $data ) {
 				return $this->renderItem( 'hero-mobile-wikiamobile', $data );
 			} elseif ( $firstInfoboxAlredyRendered ) {
 				return $this->renderItem( 'hero-mobile', $data );
+			} elseif ( $hasFeaturedVideo ) {
+				return $this->renderHeroImageAsDataItem( $data['title'], $image );
```

Comment:
@latata I would call it heroImageSmall or something like that. I think that calling it DataItem because it looks like it does not make sense, because when design changes you need to rename it everywhere. Otherwise you'll end up with misleading name